src/app/filter/filter.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, EventEmitter, Output, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'app-filter',
+  templateUrl: './filter.component.html',
+  styleUrls: ['./filter.component.scss'],
+  encapsulation: ViewEncapsulation.None
```

Comment:
why? 🤔

Reply:
I was doing some tests and I forgot to remove it